pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/stats/MetadataStoreStats.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.metadata.impl.stats;
+
+import io.prometheus.client.Counter;
+import io.prometheus.client.Histogram;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public final class MetadataStoreStats implements AutoCloseable {
+    private static final double[] BUCKETS = new double[]{1, 3, 5, 10, 20, 50, 100, 200, 500, 1000, 2000, 5000};
+    private static final String LABEL_NAME = "metadata_store_name";
+
+    private static final Histogram GET_OPS_SUCCEED = Histogram
+            .build("pulsar_metadata_store_get_ops_latency", "-")
```

Comment:
It's better to create a prefix `pulsar_metadata_store_`,

and can we remove "_ops_"? Just keep the name simple.

Reply:
> It's better to create a prefix pulsar_metadata_store_,

@tjiuming Could you please confirm if this one looks good to you? I don't see the update and comment here.

Reply:
@codelipenghui I think it's doesn't matter. do you mean like this ?
```
final String prefix = "pulsar_metadata_store_";
Histogram GET_OPS_SUCCEED = Histogram
            .build( prefix + "get_latency", "-");
Histogram DEL_OPS_SUCCEED = Histogram
            .build( prefix + "del_latency", "-");
```

Reply:
Yes, this looks very intuitive that all the metadata-related metrics start with `pulsar_metadata_store_`

Reply:
@codelipenghui resolved, PTAL